packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js

```diff
@@ -144,6 +145,10 @@ const Template = (args, { argTypes }) => ({
       <template slot="desktop-navigation-item">
         <SfLink href="#">{{item}}</SfLink>
       </template>
+      <template slot="mobile-navigation-item">
```

Comment:
Why do we want `SfMenuItem`? Any specific reason? If nothing special, please use `SfLink` is enough